mfbt/RecordReplay.h

```diff
@@ -350,6 +352,7 @@ const char* GetBuildId();
 void OnTestCommand(const char* aString);
 void OnRepaintNeeded(const char* aWhy);
 bool IsTearingDownProcess();
+already_AddRefed<nsIStreamListener> WrapNetworkStreamListener(nsIStreamListener* listener);
```

Comment:
I know the lack of comments on the methods above this is pretty lame, but if you can add one on this method that would be great.